<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HEBB/PERCEPTRON</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Algoritmo:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbAlg">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>120</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>61</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Treino:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>381</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>63</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Entradas:</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>10</y>
       <width>20</width>
       <height>311</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>63</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Saida:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnTreino">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>361</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Treinar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>360</y>
       <width>63</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;w1:&lt;/p&gt;&lt;p&gt;w2:&lt;/p&gt;&lt;p&gt;wb:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPesos">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>360</y>
       <width>63</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEntradas">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtSaida">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>40</y>
       <width>161</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>60</y>
      <width>20</width>
      <height>531</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>60</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Executar:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>100</y>
      <width>131</width>
      <height>491</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Entrada:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnExecutar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>111</width>
       <height>34</height>
      </rect>
     </property>
     <property name="text">
      <string>Executar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>63</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Saida:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblSaida">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>63</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTextEdit" name="txtEntrada">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>111</width>
       <height>271</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
